tasks/initialise.yml

```diff
@@ -22,16 +22,35 @@
         PGSETUP_INITDB_OPTIONS: >-
           --encoding=UTF8 --locale=en_US.UTF-8 --auth-host=md5
 
+    - name: postgres | Check for presence of "Modified by ome postgresql ansible role" in file
+      ansible.builtin.lineinfile:
+        path: "{{ postgresql_dist_confdir }}/postgresql.conf"
+        line: "#Modified by ome postgresql ansible role"
+        state: absent
+      check_mode: yes
+      changed_when: false
+      register: config_file_changed
```

Comment:
Primary downside of this logic is that once the configuration file has been create from the combination of the original package file and the Jinja2  template, this will also be flagged as changed.

If the PSQL variables are updated e.g. via `postgresql_server_conf`, the tasks below will be skipped when re-executing the role